students/ghassan/lesson04/trig.py

```diff
@@ -2,16 +2,28 @@
 
 from pprint import pprint
 
-with open('data.txt', 'r') as datafh:
-    data = datafh.read().split()
+''' with open('data.txt', 'r') as datafh:
+    data = datafh.read().split() '''
 
+data = "I wish I may I wish I might".split()
 z = {}
 
 
 for item in range(0, len(data)-2):
     z.setdefault('{} {}'.format(data[item], data[item+1]), [])
-    z['{} {}'.format(data[item], data[item+1])].append('{}'.format(data[item+2]))
+    z['{} {}'.format(
+        data[item], data[item+1])].append('{}'.format(data[item+2]))
 
-pprint(z)
+key = 'I may'
 
+new_file = [x for x in key.split()]
 
+
+try:
+    while True:
+        for word in range(0, len(new_file)-1):
+            for y in z[' '.join(new_file[-2:])]:
+                new_file.append(y)
```

Comment:
(correct me if I"m wrong) but it looks like it is not selecting one word from options for the key:
for example: `'I wish': ['I', 'I']` both "I" and "I" get appended to new text, when one of those should be chosen (at random)